src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -59,6 +59,7 @@
     static final String ERR_MISSING_FILE_FLAG = "Must use one of these flags: -f | -d";
     static final String ERR_TOO_MANY_FILE_FLAGS = "Can only use one of these flags: -f | -d";
     static final String ERR_SMALLER_THAN_MIN = "%s must be at least %d";
+    static final String ERR_DEPTH_REQUIRES_T = "--depth requires -t";
```

Comment:
If run with -t=1 and --depth, it does error out with this message. Probably does not need to be addressed though.

Reply:
True... `-t` is smart enough to run single-threaded validator if `t=1`.  I went ahead and tweaked the error message.